### DISCUSSION TITLE
Unit 4 is live!

### DISCUSSION THREAD

**Discussion by rasbt** [Lightning-AI/dl-fundamentals#18] [Announcements]
I am happy to share that Unit 4 of my free Deep Learning Fundamentals class is now live!

In this unit we are moving from softmax regression to multilayer neural networks. And we will be talking about important design considerations, like why it's important to use nonlinear activation function and initialize the networks with small, random weights.

Happy learning!

https://lightning.ai/pages/courses/deep-learning-fundamentals/training-multilayer-neural-networks-overview/logistic-regression-for-multiple-classes-part-1-5/
